src/lib/util/EnvironmentVariable.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Builds expected env variables in this format:
+ * SFDX_SCANNER.<ENGINE>.VARIABLE_NAME
+ */
+
+import { ENGINE } from "../../Constants";
+
+const PREFIX = 'SFDX_SCANNER';
+const SEPARATOR = '_';
+
+export enum CONFIG_NAME {
+	MINIMUM_TOKENS = 'Minimum_Tokens'
+}
```

Comment:
New enum to track config values that can be fetched through environment variables. I've also attempted to standardize the access in this new module.